traffic_ops/testing/api/v14/withobjs_test.go

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
I think you meant to move `withobjs.go` to this new filename, but I don't see the old one deleted?

Reply:
Oops!